contracts/contracts/FleekERC721.sol

```diff
@@ -536,4 +543,108 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
             delete _apps[tokenId];
         }
     }
+
+    /*//////////////////////////////////////////////////////////////
+        ACCESS CONTROL
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @dev Grants the collection role to an address.
+     *
+     * Requirements:
+     *
+     * - the caller should have the collection role.
+     *
+     */
+    function grantCollectionRole(Roles role, address account) public whenNotPaused requireCollectionRole(Roles.Owner) {
+        _grantCollectionRole(role, account);
+    }
+
+    /**
+     * @dev Grants the token role to an address.
+     *
+     * Requirements:
+     *
+     * - the caller should have the token role.
+     *
+     */
+    function grantTokenRole(
+        uint256 tokenId,
+        Roles role,
+        address account
+    ) public whenNotPaused requireTokenRole(tokenId, Roles.Owner) {
+        _grantTokenRole(tokenId, role, account);
+    }
+
+    /**
+     * @dev Revokes the collection role of an address.
+     *
+     * Requirements:
+     *
+     * - the caller should have the collection role.
+     *
+     */
+    function revokeCollectionRole(Roles role, address account) public whenNotPaused requireCollectionRole(Roles.Owner) {
+        _revokeCollectionRole(role, account);
+    }
+
+    /**
+     * @dev Revokes the token role of an address.
+     *
+     * Requirements:
+     *
+     * - the caller should have the token role.
+     *
+     */
+    function revokeTokenRole(
+        uint256 tokenId,
+        Roles role,
+        address account
+    ) public whenNotPaused requireTokenRole(tokenId, Roles.Owner) {
+        _revokeTokenRole(tokenId, role, account);
```

Comment:
Is these section going to be affected by the acl refactor?

Reply:
Yes. I will need to fix #114 after this got merged.